src/NUnitFramework/framework/Constraints/DictionaryContainsKeyValuePairConstraint.cs

```diff
@@ -41,7 +40,7 @@ public DictionaryContainsKeyValuePairConstraint(object key, object? value)
         private bool Matches(object? actual)
         {
             if (actual is null)
-                throw new ArgumentException("Expected: IDictionary But was: null", nameof(actual));
+                throw new ArgumentException("Expected: IEnumerable<KeyValuePair<,>> But was: null", nameof(actual));
```

Comment:
It also works for just `IDictionary`. 
So suggest to make it:
```suggestion
                throw new ArgumentException("Expected: IDictionary or IEnumerable<KeyValuePair<,>> But was: null", nameof(actual));
```